ee/codegen_integration/fixtures/simple_merge_node/code/workflow.py

```diff
@@ -0,0 +1,17 @@
+from vellum.workflows import BaseWorkflow
+
+from .nodes.final_output import FinalOutput
+from .nodes.merge_node import MergeNode
+from .nodes.templating_node_1 import TemplatingNode1
+from .nodes.templating_node_2 import TemplatingNode2
+from .nodes.templating_node_3 import TemplatingNode3
+
+
+class Workflow(BaseWorkflow):
+    graph = {
+        TemplatingNode2 >> MergeNode >> TemplatingNode3 >> FinalOutput,
+        TemplatingNode1 >> MergeNode >> TemplatingNode3 >> FinalOutput,
+    }
```

Comment:
This graph is currently incorrect. My next step is to update codegen to produce the correct graph, which should be:

```
graph = {
        TemplatingNode2
        TemplatingNode1,
    } >> MergeNode >> TemplatingNode3 >> FinalOutput,
```